src/types/me.ts

```diff
@@ -1,2 +1,2 @@
 export {MeState} from 'src/me/reducers'
-export {AccountType, Me} from 'src/client/unityRoutes'
+export {AccountType} from 'src/client/unityRoutes'
```

Comment:
updating the openapi sha brought in another [change](https://github.com/influxdata/openapi/commit/f6ad4d26c7a4176fec4b625c7dfe236067bec73d) and removing this import is required